pywis_pubsub/publish.py

```diff
@@ -99,6 +105,8 @@ def create_message(topic: str, content_type: str, url: str, identifier: str,
     :identifier: `str` of unique-id to help global broker deduplicate data
     :geometry: point array defining longitude,latitude,elevation
                (elevation is optional
+    :inline: `bool` of whether to publish the data inline as base64
```

Comment:
This comment is slightly misleading, the data will only be published inline if this boolean is true -and- the data-size is less than 4096 bytes